client/app/components/EditParameterSettingsDialog.jsx

```diff
@@ -18,8 +20,16 @@ function getDefaultTitle(text) {
   return capitalize(words(text).join(' ')); // humanize
 }
 
-function NameInput({ name, onChange, existingNames, setValidation }) {
-  let helpText = `This is what will be added to your query editor {{ ${name} }}`;
+function isTypeDate(type) {
+  return startsWith('date', type) && !isTypeDateRange(type);
+}
+
+function isTypeDateRange(type) {
+  return /-range/.test(type);
```

Comment:
Doesn't seem to work at all 😳 

![image](https://user-images.githubusercontent.com/71468/53861960-72111800-3fee-11e9-91fe-227032a9aa4e.png)

Reply:
It's only for keyword - not title

Reply:
🤦‍♂️